File: js/lv1/문자열내림차순으로배치하기.js
function solution(s) {
  return s.split("").sort().reverse().join("");
}

strings = ["Zbcdefg"];
answer = ["gfedcbZ"];

for (let i = 0; i < strings.length; i++) {
  if (solution(strings[i]) == answer[i]) {
    console.log(`${i + 1}: pass`);
  } else {
    console.log(`${i + 1}: fail`);
  }
}

File: python/lv1/문자열내림차순으로배치하기.py
def solution(s):
    return ''.join(sorted(s, reverse=True))


strings = ['Zbcdefg']
answer = ['gfedcbZ']

for i in range(len(strings)):
    if solution(strings[i]) == answer[i]:
        print(f'{i+1}: pass')
    else:
        print(f'{i+1}: fail')